include/spirv-tools/libspirv.h

```diff
@@ -375,6 +379,21 @@ spv_context spvContextCreate(spv_target_env env);
 // Destroys the given context object.
 void spvContextDestroy(spv_context context);
 
+// Creates a Validator options object with default options.
+spv_validator_options spvValidatorOptionsCreate();
+
+// Destroys the given Validator options object.
+void spvValidatorOptionsDestroy(spv_validator_options options);
+
+// Records the maximum number of structure members that is considered valid in
```

Comment:
Each new limit would imply another entry point.  We can do better by having a single "set limit" method, which takes an enum.

Follow the example in shaderc: https://github.com/google/shaderc/blob/master/libshaderc/include/shaderc/shaderc.h#L345 

The only question is what type the number should have. Either int or uint32_t.  Shaderc went with int.
The universal limits table in the spec shows that a 32 bit signed int is sufficient.  I could go either way.
So far we have no use for negative limits, so I'd probably lean toward uint32_t.
If we ever need a 64bit limit then we can add another entry point.